RadioIDs.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from typing import Dict, List
+
+log = logging.getLogger(__name__)
+namecache: Dict[str, str] = {}
+
+
+def _getSet(srcList: List[dict]) -> List[str]:
+    """Generates a list of unique sources.
+
+    Args:
+        srcList (List[dict]): The srcList to process.
+
+    Returns:
+        List[str]: Unique source IDs.
+    """
+    result = set()
+    for src in srcList:
+        result.add(src["src"])
+    return list(result)
```

Comment:
FWIW this can be simplified:
```python
return list({src["src"] for src in srcList})
```
The brackets are a "set comprehension".
Example:
```python
[ins] In [1]: a = [{"a": 1}, {"a": 2}, {"a": 1}]

[ins] In [2]: list({x["a"] for x in a})
Out[2]: [1, 2]
```